bin/run_umt.sh

```diff
@@ -97,9 +86,6 @@ if [ "$1" == "build_umt" ]; then
             -DMPI_CXX_COMPILER=${MPI_PATH}/bin/mpicxx \
             -DMPI_Fortran_COMPILER=${MPI_PATH}/bin/mpifort \
             -DCMAKE_INSTALL_PREFIX=${UMT_INSTALL_PATH} \
-    		-DMETIS_ROOT=${UMT_INSTALL_PATH} \
-            -DHYPRE_ROOT=${UMT_INSTALL_PATH} \
-    		-DMFEM_ROOT=${UMT_INSTALL_PATH} \
```

Comment:
Where these software packages simply not needed for our testing purposes or are these now picked up from some system-level installation that may break in the future?

Reply:
The dependency on MFEM is now optional. We can build UMT without it. MFEM hast introduced the dependencies on the Hyper and Metis projects.